measures/2024/measures-data.json

```diff
@@ -15209,7 +15209,7 @@
     "isHighPriority": true,
     "isInverse": false,
     "isRiskAdjusted": false,
-    "primarySteward": "ABG QCDR in Collaboration with Insight Medical Data Service, LLC",
+    "primarySteward": "ABG QCDR in Collaboration with Insight Medical Data Services, LLC in Collaboration with Insight Medical Data Services, LLC",
```

Comment:
This was incorrect in the file - this should just read "ABG QCDR in Collaboration with Insight Medical Data Services, LLC"